python/cuml/test/test_random_forest.py

```diff
@@ -443,7 +444,7 @@ def test_rf_classification_sparse(small_clf, datatype,
         tl_model = cuml_model.convert_to_treelite_model()
         assert num_treees == tl_model.num_trees
         assert X.shape[1] == tl_model.num_features
-        del tl_model
+        # del tl_model
```

Comment:
Was this left intentially from a double free?

Reply:
Oops, should go. Deleted it.